src/Search.js

```diff
@@ -0,0 +1,22 @@
+import React, { useRef, useEffect, useState } from 'react';
+// import './search.css';
+
+export default function Search() {
+  const [searchValue, setSearchValue] = useState("");
+  const inputRef = useRef();
+
+  useEffect(() =>{
+    inputRef.current.focus();
+  }, []);
+  
+  function handleChange(e) {
+    setSearchValue(e.target.value);
+  }
+
```

Comment:
I don't have much to add. The component is looking great. As I mentioned above, the only thing that is missing is the search logic itself. So I would change the idea of the package to search input component.